pkg/util/util.go

```diff
@@ -301,3 +302,16 @@ func WaitUntilTimeout(name string, fn func(), timeout time.Duration) {
 	case <-exited:
 	}
 }
+
+// ParseGCDuration parses gc durations. The default unit is day.
+func ParseGCDuration(gc string) (time.Duration, error) {
+	d, err := strconv.ParseUint(gc, 10, 64)
+	if err == nil {
+		return time.Duration(d) * 24 * time.Hour, nil
+	}
+	gcDuration, err := time.ParseDuration(gc)
+	if err != nil {
+		return 0, errors.Errorf("unsupported gc time %s, please use 7 or 8h(max unit is hour) format gc time. err: %s", gc, err)
```

Comment:
errors.Annotatef(err, "unsupported gc time %s, etc: use 7 for 7day,  7h for 7 hour")